packages/db/package.json

```diff
@@ -38,6 +38,7 @@
     "@chainsafe/ssz": "^0.15.1",
     "@lodestar/config": "^1.18.1",
     "@lodestar/utils": "^1.18.1",
+    "classic-level": "^1.4.1",
```

Comment:
It might make sense to depend on `classic-level` dependency directly now that we are requiring it.

Reply:
Right, the code already expect running under NodeJS and we add `classic-level` dependency to make sure the right version is used.

Reply:
Sounds good to me.
Can you update this PR to include that? Currently it feels like in a halfway state (which I suppose caused your comment here).